packages/shared/src/relay/missingFieldHandlers.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render } from '@testing-library/react';
+import nock from 'nock';
+import React from 'react';
+import { RelayEnvironmentProvider, useLazyLoadQuery } from 'react-relay';
+import { fetchQuery, graphql, Network, RecordSource, Store } from 'relay-runtime';
+import RelayModernEnvironment from 'relay-runtime/lib/store/RelayModernEnvironment';
+
+import { missingFieldHandlersTestFirstQuery } from '~/generated/missingFieldHandlersTestFirstQuery.graphql';
+import { missingFieldHandlersTestSecondQuery } from '~/generated/missingFieldHandlersTestSecondQuery.graphql';
+import { createMissingFieldHandlers } from '~/relay/missingFieldHandlers';
+import { createRelayFetchFunction } from '~/relay/network';
+
+describe('missingFieldHadlers', () => {
```

Comment:
This is a pretty sweet test. It ensures `galleryById` resolves without a network call even though that resolver has never been in the cache.

Reply:
this is indeed a sweet test 🔥🔥🔥🔥🔥 

thanks for writing this man

Reply:
small typo in `missingFieldHadlers`